### ISSUE TITLE
Fix aria-describedby for subscribe form text fields 

### ISSUE THREAD

**Issue by padms** [equinor/energyvision#666]
https://github.com/equinor/design-system/issues/1145 
It is not possible currently to use aria-describedby for the EDS Textfield. Once the above issue is fixed, verify the fields in subscribe form for accessibility.

**Comment by padms** [comment]
Updated eds-core version

**Comment by wenche** [comment]
@padms  I removed the aria-invalid attribute :thumbsup:

Otherwise LGTM! The EDS team has tested this by using a screen reader, and found that the helper text is now read. And it should be OK to have an empty or undefined aria-describedby, they are in that case just ignored.

**Comment by wenche** [comment]
(I think it's hard for us to verify 100% that this is the most optimal solution for screen readers,  but if/when we have a dedicated a11y person to test the forms should be on the list). Closing this.
